src/index.html

```diff
@@ -3,11 +3,42 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="card">
+      <div class="card__image">
+        <img src="./images/imac.jpeg" alt="imac">
+      </div>
+      <div class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__row card__row--top">
+        <div class="stars stars--4">
```

Comment:
```suggestion
      <div class="card__image">
        <img src="./images/imac.jpeg" alt="imac">
      </div>
      
      <div class="card__text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
      <div class="card__code">Product code: 195434</div>
      
      <div class="card__row card__row--top">
        <div class="stars stars--4">
```
checklist point
<img width="949" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/102411130/1f31a5fe-d4cf-477f-89ce-7b17d5a8282b">